parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -89,23 +90,25 @@ fi
 time_metric "InstallContainerRuntime" installContainerRuntime
 {{end}}
 
+{{- if NeedsContainerd}}
+time_metric "InstallContainerd" installContainerd
+{{end}}
+
 if [[ -n "${MASTER_NODE}" ]] && [[ -z "${COSMOS_URI}" ]]; then
     {{- if IsDockerContainerRuntime}}
     CLI_TOOL="docker"
     {{else}}
     CLI_TOOL="img"
+    # This codepath requires img, which is not installed until
```

Comment:
You can remove this comment (though it was useful for my code review!)

Reply:
Done